lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -37,21 +41,20 @@ const rule = (primary) => {
 		}
 
 		root.walkAtRules(/^media$/i, (atRule) => {
-			/** @type {Array<{tokens(): Array<import('@csstools/css-tokenizer').CSSToken>}>} */
-			let invalidNodes = [];
+			const atRuleParamIndexValue = atRuleParamIndex(atRule);
 
 			parseMediaQuery(atRule).forEach((mediaQuery) => {
 				if (isMediaQueryInvalid(mediaQuery)) {
 					// Queries that fail to parse are invalid.
-					invalidNodes.push(mediaQuery);
+					complain(atRule, atRuleParamIndexValue, mediaQuery);
```

Comment:
question

Can we lint files in separate threads?
If not just resolve this comment.
I wonder if other reports from other rules could insert themselves in between the reports of this rule.
i.e. unlike before the reports are not grouped